tests/directive/max.spec.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ Copyright 2013-Present the original author or authors from the JHipster project.
```

Comment:
@Dufgui can you please rename the file to `max.directive.spec.ts` same for other file